build.sbt

```diff
@@ -31,11 +33,15 @@ lazy val root = project
   .aggregate(baseJVM, baseJS, metaJVM, metaJS, stdJVM, stdJS, microsite, benchmarks)
   .enablePlugins(ScalaJSPlugin)
 
-resolvers in ThisBuild += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
+lazy val zioJVM = ProjectRef(uri("git://github.com/scalaz/scalaz-zio.git#master"), "coreJVM")
```

Comment:
Isn't pointing to `master` equivalent to depending on the snapshot?
Should we change it to a commit SHA until we start tagging?

Reply:
Not quite the same, but close enough. It was a placeholder since I don't have a good project intuition. I can change it to the current sha if that is desired.

Reply:
I think current SHA would do it by now. Mostly to avoid what happened that you had to fix. We can bump it from time to time.
The other option is to point to master until it breaks and only then we fix the ref to the last working commit while we fix it.

Reply:
In a perfect world I think it would track the repository head until the tests fail here, at which point it fixes on the last working commit until new ones are passing again. The git/github/travis machinery is probably way too coarse to allow such a flow. I'll inject the current sha.